docs/howtoguides/how_to_collect_logs.md

```diff
@@ -1,21 +1,27 @@
 # How to collect Ubuntu Pro Client logs
 
-To collect all of the necessary logs for Ubuntu Pro Client (`pro`), please run the command:
+To collect all of the necessary logs for Ubuntu Pro Client (`pro`), please run
+the command:
 
 ```console
 $ sudo pro collect-logs
 ```
 
-This command creates a tarball with all relevant data for debugging possible problems with `pro`.
+This command creates a tarball with all relevant data for debugging possible
+problems with `pro`.
+
 It puts together:
-* The Ubuntu Pro Client configuration file (the default is `/etc/ubuntu-advantage/uaclient.conf`)
+
+* The Ubuntu Pro Client configuration file (the default is
+  `/etc/ubuntu-advantage/uaclient.conf`)
 * The Ubuntu Pro Client log files (the default is `/var/log/ubuntu-advantage*`)
-* The files in `/etc/apt/sources.list.d/*` related to UA
+* The files in `/etc/apt/sources.list.d/*` related to Ubuntu Advantage
```

Comment:
```suggestion
* The files in `/etc/apt/sources.list.d/*` related to Ubuntu Pro
```

Reply:
Good catch! Thanks Grant :) I'm still a bit woolly on exactly what the naming convention currently is, but that's cleared it up